tools/raisinbread_refresh.php

```diff
@@ -133,7 +133,7 @@
         escapeshellarg($host),
         escapeshellarg($password)
     );
-    exec($mysqlCommand . ' -e "show tables;" 2>&1 1>/dev/null', $output, $status);
+    exec($mysqlCommand . ' -e "show tables;" 1>/dev/null 2>&1', $output, $status);
     print_r($output);
```

Comment:
Can you delete this line? It shouldn't be here. Or it should at least be checked only in the if statement below.

Reply:
good catch, this remove the annoying "array" output.